index.html

```diff
@@ -1,59 +1,225 @@
+							Gain+Glory
+						</h3>
+						<ul class="cd-links">
+							<li><a href="">Ruby on Rails</a></li>
+							<li><a href="">css</a></li>
+							<li><a href="">Javascript</a></li>
+							<li><a href="">html</a></li>
+						</ul>
+						<button>See Project</button>
+					</div>
+				</div>
+				<div class="card cd2">
+					<div class="img-hdr"><img src="" alt="" /></div>
+					<div class="langs">
+						<h3>
+							Multi-Post Stories <br />
+							Gain+Glory
+						</h3>
+						<ul class="cd-links">
+							<li><a href="">Ruby on Rails</a></li>
+							<li><a href="">css</a></li>
+							<li><a href="">Javascript</a></li>
+							<li><a href="">html</a></li>
+						</ul>
+						<button>See Project</button>
+					</div>
+				</div>
+				<div class="card cd3">
+					<div class="img-hdr"><img src="" alt="" /></div>
+					<div class="langs">
+						<h3>
+							Multi-Post Stories <br />
+							Gain+Glory
+						</h3>
+						<ul class="cd-links">
+							<li><a href="">Ruby on Rails</a></li>
+							<li><a href="">css</a></li>
+							<li><a href="">Javascript</a></li>
+							<li><a href="">html</a></li>
+						</ul>
+						<button>See Project</button>
+					</div>
+				</div>
+				<div class="card cd4">
+					<div class="img-hdr"><img src="" alt="" /></div>
+					<div class="langs">
+						<h3>
+							Multi-Post Stories <br />
+							Gain+Glory
+						</h3>
+						<ul class="cd-links">
+							<li><a href="">Ruby on Rails</a></li>
+							<li><a href="">css</a></li>
+							<li><a href="">Javascript</a></li>
+							<li><a href="">html</a></li>
+						</ul>
+						<button>See Project</button>
+					</div>
+				</div>
+				<div class="card cd5">
+					<div class="img-hdr"><img src="" alt="" /></div>
+					<div class="langs">
+						<h3>
+							Multi-Post Stories <br />
+							Gain+Glory
+						</h3>
+						<ul class="cd-links">
+							<li><a href="">Ruby on Rails</a></li>
+							<li><a href="">css</a></li>
+							<li><a href="">Javascript</a></li>
+							<li><a href="">html</a></li>
+						</ul>
+						<button>See Project</button>
+					</div>
+				</div>
+				<div class="card cd6">
+					<div class="img-hdr"><img src="" alt="" /></div>
+					<div class="langs">
+						<h3>
+							Multi-Post Stories <br />
+							Gain+Glory
+						</h3>
+						<ul class="cd-links">
+							<li><a href="">Ruby on Rails</a></li>
+							<li><a href="">css</a></li>
+							<li><a href="">Javascript</a></li>
+							<li><a href="">html</a></li>
+						</ul>
+						<button>See Project</button>
+					</div>
+				</div>
+			</div>
+		</section>
+
+		<section id="about">
+			<div class="about-des">
+				<div class="about-ttl">
+					<h1>About me</h1>
+				</div>
+				<div class="about-para">
+					<p>
+						I can help you build a product , feature or website Look through
+						some of my work and experience! If you like what you see and have a
+						project you need coded, don’t hestiate to contact me.
+					</p>
+					<button>Get My Resume</button>
```

Comment:
- [ ] As stated the project requirement Kindly Implement the button interactions follow the button styling: Active, Hover and Pressed (see design).

![Capture](https://user-images.githubusercontent.com/93598189/212038100-cbfbf329-e603-4e98-a910-5619b51b0295.PNG)